launch_testing_ament_cmake/CMakeLists.txt

```diff
@@ -14,8 +14,8 @@ install(
 )
 
 if(BUILD_TESTING)
-  find_package(ament_cmake_pytest REQUIRED)
-  ament_add_pytest_test(launch_testing_ament_cmake_pytests test)
```

Comment:
Why should this test be removed?

Reply:
The only test that was in there was test_copyright. The copyright check is now done by `ament_copyright()` instead.